<?xml version="1.0"?>
<launch>

  <arg name="use_gazebo" value="true" />

  <!-- <include file="$(find moveit_config)/launch/enviorment.launch">
    <arg name="use_gazebo" value="$(arg use_gazebo)"/>
  </include> -->
  
  <node name="arm_commander" pkg="ob1_arm_control" type="arm_commander.py"  respawn="false" output="screen"/>
</launch>